agent-server/model/npc.go

```diff
@@ -1,14 +1,12 @@
 package model
 
 import (
-	"github.com/ferdoran/go-sro/agent-server/navmeshv2"
 	"github.com/sirupsen/logrus"
 	"sync"
 )
 
 type NPC struct {
 	SRObject
-	navmeshv2.RtNavmeshPosition
```

Comment:
Looks like `navmeshv2.RtNavmeshPosition` is already included in `SRObject` and the position is set/written on `NPC` but during packet creation in [agent-server/model/packeturils_entity](https://github.com/ferdoran/go-sro/blob/887cabaf45414e98492ddd0cc7bd62e2d3a799c5/agent-server/model/packetutils_entity.go#L52) it is read from `SRObject`. 

I think having it in `SRObject` only makes sense.

Good spot!